app/containers/scenes/ProfileScene.js

```diff
@@ -1,24 +1,90 @@
 import React, { Component } from 'react';
 import {
   View,
-  Text
+  Text,
+  TextInput,
+  Image,
+  TouchableHighlight
 } from 'react-native';
 
+import { ProfileSceneCss as styles } from './ProfileScene.css.js'
+
 export default class ProfileScene extends Component {
 
   constructor(props) {
-    super(props);
+    super(props)
+    let data = JSON.parse('{"id":"0a9ds7a767ac7vr5as4de4465464646","firstName":"Tina","lastName":"Turner","displayName":"Tina Turner","email":"tina@singer.com","birthday":"Aug 20, 1998","interestedChallenges":["d53h338383hjsjbs623yjb923u2h2939","9a7a5d5d7v8a7hwh11199dhsdjb2387y"],"completedChallenges":["6a6a5a55sdfh12j3jb1423123412jhkjs"],"createdChallenges":["6a6a5a55sdfh12j3jb1423123412jhkjs"],"created":"Mon Feb 06 2017 00:45:48 GMT-0700 (MST)"}')
+    this.state = {
+      userData: data,
+      completedCount: data.completedChallenges.length,
+      createdCount: data.createdChallenges.length,
+      profChar: data.firstName.charAt(0),
+      edit: false
+    }
   }
 
-  goBack() {
-    this.props.navigator.pop();
+  // TODO: load profile image into circle,
+  // TODO: have default but maybe customizable image behind
+
+  editDisplayName() {
+    this.setState(prevState => ({
```

Comment:
This function can be called as:
1) `this.setState({key: 12 }`
2) `this.setState(prevState => { return {key: prevState.key + 1} })`
3)  `this.setState((prevState, props) => { return {key: prevState.key + props.increment} })`
Since you aren't using prevState or props, I would also recommend removing prevState and just passing an object directly to the function as in the first example.